src/scripts/main.js

```diff
@@ -1 +1,66 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+function notification(className, message) {
+  body.insertAdjacentHTML('beforeend', `
+  <div data-qa="notification" class="${className}">${message}</div>
+  `);
+};
+
+const firstPromise = new Promise((resolve, reject) => {
+  let clicked = false;
+
+  body.addEventListener('click', (e) => {
+    if (e.button === 0) {
+      clicked = true;
+      resolve('First promise was resolved');
+    }
+  });
+
+  setTimeout(() => {
+    if (!clicked) {
+      reject(new Error('First promise was rejected'));
+    }
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  document.body.addEventListener('click', (e) => {
+    if (e.button === 0 || e.button === 2) {
+      resolve('Second promise was resolved');
+    }
+  });
```

Comment:
not fixed
here should be listener also for 'contextmenu' event
something like that
```
   document.addEventListener('contextmenu', (e) => {
    e.preventDefault();
    resolve('Second promise was resolved');
  });